### ISSUE TITLE
Allow customization of permissions

### ISSUE THREAD

**Issue by nickspencer2** [nickspencer2/chord#4]
Allow users to set up custom permissions for commands (perhaps allow group names and IDs), e.g. 

```json
"play": {
    "permissions": ["any", 123123123123]
}
```

Also allow setting up custom groups such as 

```json
"permissions": {
    "notquitemods": [123123123, 234234234]
}
```

**Comment by nickspencer2** [comment]
Created a branch for this in branch custom-permissions. Development not yet started.

**Comment by nickspencer2** [comment]
Resolved in #5
